pulsar/README.md

```diff
@@ -0,0 +1,86 @@
+# Pulsar Bindings
+This document defines how to describe Apache Pulsar specific information with AsyncAPI.
+
+<a name="version"></a>
+
+## Version
+
+Current version is `0.1.0`.
+
+<a name="server"></a>
+## Server Binding Object
+
+This object contains information about the server representation in Pulsar.
+
+##### Fixed Fields
+
+Field Name | Type | Description |
+---|:---:|:---:|
+`retention` | [Retention Definition Object](#retention-definition-object) | Topic retention policy  |
```

Comment:
Perhaps adding the Pulsar Tenant here would be a nice addition, as it cannot really be inferred from the Server.Url